modules/ingest-user-agent/src/main/java/org/elasticsearch/ingest/useragent/IngestUserAgentPlugin.java

```diff
@@ -53,9 +53,12 @@
 
     static Map<String, UserAgentParser> createUserAgentParsers(Path userAgentConfigDirectory, UserAgentCache cache) throws IOException {
         Map<String, UserAgentParser> userAgentParsers = new HashMap<>();
+        InputStream deviceTypeRegexStream = IngestUserAgentPlugin.class.getResourceAsStream("/device_type_regexes.yml");
+
 
         UserAgentParser defaultParser = new UserAgentParser(DEFAULT_PARSER_NAME,
-                IngestUserAgentPlugin.class.getResourceAsStream("/regexes.yml"), cache);
+            IngestUserAgentPlugin.class.getResourceAsStream("/regexes.yml"),
+            deviceTypeRegexStream, cache);
```

Comment:
Minor simplification:
```suggestion
            IngestUserAgentPlugin.class.getResourceAsStream("/device_type_regexes.yml"), 
            cache);
```
and remove the earlier declaration on line 56.